templates/TOPStyleGuide.md

```diff
@@ -216,18 +223,16 @@ specific file:
 Be sure to update your `README.md`!
 ```
 
-Backticks are the most common approach for "escaping" Markdown metacharacters;
-in most situations where escaping would be needed, code font just makes sense
-anyway.
-
 ### Codeblocks
 
 For code quotations longer than a single line, use a codeblock:
 
 <pre>
-```python
-def Foo(self, bar):
-  self.bar = bar
+```javascript
```

Comment:
Can we change the three back ticks to three tilde's please 
```
~~~javascript
```

The parser we use on the site expects those instead of backticks

Reply:
What parser does TOP use?

Reply:
I believe it's kramdown.